### ISSUE TITLE
Plotting of multigraphs fails with Cairo backend

### ISSUE THREAD

**Issue by szhorvat** [igraph/python-igraph#474]
**Describe the bug**

Plotting of graphs with multi-edges fails with the Cairo backend.

There should be a test in the test suite for this.

**To reproduce**

Use the Cairo backend.

```
import igraph as ig

ig.plot(ig.Graph.Formula('a-b-c-a-b', simplify=False))
```

<details>
<summary>Output</summary>

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~/anaconda/envs/igdev/lib/python3.9/site-packages/IPython/core/formatters.py in __call__(self, obj)
    343             method = get_real_method(obj, self.print_method)
    344             if method is not None:
--> 345                 return method()
    346             return None
    347         else:

~/anaconda/envs/igdev/lib/python3.9/site-packages/igraph-0.9.8-py3.9-macosx-10.9-x86_64.egg/igraph/drawing/cairo/plot.py in _repr_svg_(self)
    329         context = cairo.Context(surface)
    330         # Plot the graph on this context
--> 331         self.redraw(context)
    332         # No idea why this is needed but python crashes without
    333         context.show_page()

~/anaconda/envs/igdev/lib/python3.9/site-packages/igraph-0.9.8-py3.9-macosx-10.9-x86_64.egg/igraph/drawing/cairo/plot.py in redraw(self, context)
    281                 else:
    282                     ctx.save()
--> 283                 plotter("cairo", ctx, bbox=bbox, palette=palette, *args, **kwds)
    284                 if opacity < 1.0:
    285                     ctx.pop_group_to_source()

~/anaconda/envs/igdev/lib/python3.9/site-packages/igraph-0.9.8-py3.9-macosx-10.9-x86_64.egg/igraph/drawing/graph.py in __plot__(self, backend, context, *args, **kwds)
    556         DrawerDirectory.resolve(self, backend)(context),
    557     )
--> 558     drawer.draw(self, *args, **kwds)

~/anaconda/envs/igdev/lib/python3.9/site-packages/igraph-0.9.8-py3.9-macosx-10.9-x86_64.egg/igraph/drawing/cairo/graph.py in draw(self, graph, *args, **kwds)
    271             src, dest = edge.tuple
    272             src_vertex, dest_vertex = vertex_builder[src], vertex_builder[dest]
--> 273             drawer_method(visual_edge, src_vertex, dest_vertex)
    274 
    275         # Construct the iterator that we will use to draw the vertices

~/anaconda/envs/igdev/lib/python3.9/site-packages/igraph-0.9.8-py3.9-macosx-10.9-x86_64.egg/igraph/drawing/cairo/edge.py in draw_undirected_edge(self, edge, src_vertex, dest_vertex)
    105         if edge.curved:
    106             (x1, y1), (x2, y2) = src_vertex.position, dest_vertex.position
--> 107             aux1, aux2 = get_bezier_control_points_for_curved_edge(x1, y1, x2, y2, edge['curved'])
    108             ctx.curve_to(aux1[0], aux1[1], aux2[0], aux2[1], *dest_vertex.position)
    109         else:

TypeError: 'VisualEdgeBuilder.Element' object is not subscriptable
```

</details>

**Version information**

Current `develop` branch. It works fine in 0.9.8.

**Comment by iosonofabio** [comment]
Fixed in bcaa869, but leaving open until we have a test

**Comment by szhorvat** [comment]
@iosonofabio Do you want to close this now?
